R/create-test-framework.R

```diff
@@ -106,44 +86,24 @@ Call find_tests_without_reqs() with the returned data frame to see them."))
     tests <- tests[tests_is_linked, ]
   }
 
+  if(any(is.na(dd$TestName)) | any(is.na(dd$passed))){
+    missing_tests <- dd %>% filter(is.na(dd$TestName) | is.na(dd$passed))
+    warning(glue("{nrow(missing_tests)} test(s) referenced in requirements, but not found in test outputs: {paste(missing_tests$TestId, collapse = ', ')}"))
+    # dd <- dd %>% filter(!is.na(.data$passed))
+  }
+
+
```

Comment:
I had to revert the filtering here (`dd <- dd %>% filter(!is.na(.data$passed))`) to get the tests in `test-input-checks` to pass. The filtering defeated the purpose of `find_reqs_with_missing_tests()`. Its interesting that we didnt have a warning before though (at least as far as I could tell). 

I opted to change the warning to an alert, and not actually drop the tests, so that users could still take advantage of `find_reqs_with_missing_tests()`. I kept the new filter as a comment in case we decide to change the test and/or remove that helper function, but I think we should have a discussion about what makes the most sense for users